templates/meta_data/cobol.toml

```diff
@@ -0,0 +1,8 @@
+id = "cobol"
+weight = 1
+type = "boilerplate"
+title = "COBOL"
+description = "Set up a Worker written in COBOL"
```

Comment:
Could we add more description? What does the project actually do is it hello world?